Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -65,6 +68,16 @@ function GetCommentsFromIssues
     return $comments
 }
 
+function GetRuleId
```

Comment:
I'll add a comment explaining that the $issue.rule is of the form "<repository>:<ruleID>"